### PR TITLE
Rework the messaging section

### PR DESCRIPTION
**Pull request by eolivelli**

### CODE REVIEW

File: configuration-resources/messaging/pulsar.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+### Examples
+
+This is how it looks like a configuration for a Pulsar instance:
+
+```yaml
+    streamingCluster:
+    type: "pulsar"
+    configuration:
+        admin:
+            serviceUrl: "pulsar://localhost:6650"
```
Comment (nicoloboschi) [inline configuration-resources/messaging/pulsar.md:28]: shouldn't be the opposite here?

**Review by nicoloboschi — CHANGES_REQUESTED** [review]

**Review by nicoloboschi — APPROVED** [review]

File: configuration-resources/messaging/pulsar.md (hunk 1 above)
Comment (eolivelli) [reply configuration-resources/messaging/pulsar.md]: good catch

File: configuration-resources/messaging/kafka.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+### Configuration
+
+In order to connect to Kafka Broker you must provide the addess of the broker with the `bootstrap.servers` property.
+
+Please refer to the Apache Kafka Java client documentation for more details about the configuration properties.
+
+
+### Authentication
+
+Usually in Apache Kafka you provide Authentication using the JAAS configuration file, in LangStream you can provide the configuration directly in the instance.yaml file.
```
Comment (mendonk) [inline configuration-resources/messaging/kafka.md:14]:
```suggestion
Usually in Apache Kafka you would provide Authentication using the JAAS configuration file, but in LangStream you can provide the configuration directly in the instance.yaml file.
```

File: configuration-resources/messaging/pulsar.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+# Using Apache Pulsar
+
+Configuration for a LangStream streaming instance.
+
+### Configuration
+
+In order to connect to a Pulsar broken you must provide the following configuration:
```
Comment (mendonk) [inline configuration-resources/messaging/pulsar.md:7]:
```suggestion
To connect to a Pulsar broker, provide the following configuration:
```

**Review by mendonk — APPROVED** [review]
+1

**Pull request merged**